### PR TITLE
[DDO-1309] Rawls usage of Liquibase migration library chart

### PR DESCRIPTION
**Pull request by jack-r-warren**
Separate PR because this one references version numbers made by #395. That PR will eventually have to be merged before this one, so that this one can be made to reflect the final, released version number.

### CODE REVIEW

**Comment by jack-r-warren** [comment]
Better seen through the helmfile diff at https://github.com/broadinstitute/terra-helmfile/pull/1378, here: https://github.com/broadinstitute/terra-helmfile/pull/1378#issuecomment-862592584

**Review by mflinn-broad — APPROVED** [review]
lgtm once library chart is merged 👍🏻

**Pull request merged**
